executor/batch_point_get_test.go

```diff
@@ -346,3 +346,16 @@ func (s *testBatchPointGetSuite) TestPointGetForTemporaryTable(c *C) {
 	tk.MustQuery("select * from t1 where id = 1").Check(testkit.Rows("1 1"))
 	tk.MustQuery("select * from t1 where id = 2").Check(testkit.Rows())
 }
+
+func (s *testBatchPointGetSuite) TestBatchPointGetIssue25167(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t (a int)")
```

Comment:
should be
```suggestion
	tk.MustExec("create table t (a int) primary key")
```

Reply:
It is not related to the root cause, though, whatever.